opendevin/runtime/docker/ssh_box.py

```diff
@@ -738,21 +769,21 @@ def close(self):
     except Exception as e:
         logger.exception('Failed to start Docker container: %s', e)
         sys.exit(1)
-
+    logger.setLevel('DEBUG')
     logger.info(
         "Interactive Docker container started. Type 'exit' or use Ctrl+C to exit."
     )
-
-    # Initialize required plugins
-    ssh_box.init_plugins([AgentSkillsRequirement(), JupyterRequirement()])
-    logger.info(
-        '--- AgentSkills COMMAND DOCUMENTATION ---\n'
-        f'{AgentSkillsRequirement().documentation}\n'
-        '---'
-    )
+    if 1:
```

Comment:
Why?

Reply:
quick switch to disable the plugin. got error in initialize plugins

Reply:
I think we can do this in a much better way.